x/participation/module.go

```diff
@@ -0,0 +1,171 @@
+package participation
+
+import (
+	"encoding/json"
+	"fmt"
+
+	// this line is used by starport scaffolding # 1
+
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	"github.com/tendermint/spn/x/participation/client/cli"
+	"github.com/tendermint/spn/x/participation/keeper"
+	"github.com/tendermint/spn/x/participation/types"
```

Comment:
Need to goimport

Reply:
goimport wasn't doing anything so I ended up manually mirroring imports format of `launch` module